.ci/workspace-startup/che7_ephemeral.json

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
Why Che 7 devfile is not used ?

Reply:
these tests have been created a long time ago, would require some extra fixes, but I'm pretty sure it can be updated to use devfiles.

Reply:
> @tdancs could you add the issue to the next sprint for adding docs and replacing workspace config to devfile and link it to this PR ?

Absolutely :)